spec/models/scorm_package_spec.rb

```diff
@@ -12,8 +12,7 @@ def get_manifest_entry(zip)
 
 describe "ScormPackage" do
   it "should find all entries in same directory as manifest" do
-    zip = Zip::File.new("spec/fixtures/scorm_package.zip")
-    manifest = get_manifest_entry zip
+    zip, manifest = get_zip_manifest("scorm_package.zip")
```

Comment:
The thing to be careful of, here, is that `zip` will have been closed by `get_zip_fixture`. I'm not 100% sure of the semantics surrounding `Zip::File.open`, but I suspect it works similarly to `File.open`, where the file is closed when the block terminates. If you need that zip file handle to remain open, you should yield `zip` and `manifest` to a block in turn, rather than returning them directly.